src/url-params.ts

```diff
@@ -163,10 +163,20 @@ export class AnalyticsUrlParams {
 	/**
 	 * @return full query parameter string that can be appended to URLs
 	 */
-	getQueryString(): string {
-		return [this.getDeviceIdsQueryString(), this.getSessionIdQueryString()]
-			.filter(x => x)
-			.join('&');
+	getQueryString(destinationHostname?: string): string {
+		const regex = /([a-zA-Z0-9-]+)(\.[a-zA-Z]{2,5})?(\.[a-zA-Z]+$)/g;
+		const actualDomain = window.location.hostname.match(regex);
+		const destinationDomain = destinationHostname
+			? destinationHostname.match(regex)
+			: undefined;
+
+		if (actualDomain !== destinationDomain) {
+			return [this.getDeviceIdsQueryString(), this.getSessionIdQueryString()]
+				.filter(x => x)
+				.join('&');
+		} else {
+			return '';
+		}
```

Comment:
```suggestion
		return '';
```

I believe this is the standard we've been following for default returns (instead of explicit `else`).

Reply:
Fixed!